buildx.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env sh
+#shellcheck shell=sh
+
+set -xe
+
+REPO=mikenye
+IMAGE=planefinder
+PLATFORMS="linux/amd64,linux/arm/v7,linux/arm64"
```

Comment:
I'll try building this on `linux/arm/v6` too, which we can add after the PR too.